app/javascript/components/lists/ListEntry.js

```diff
@@ -11,8 +11,10 @@ const ListEntry = (props) => {
   const id = ariaControls + "-list";
   const href = "#list-" + list.id;
   const badgeItemCount = list.unbought + "/" + list.item_count;
-  const sharedName = list.shared ? " (shared)" : "";
-  const listName = list.name + sharedName;
+  const sharedName = list.is_shared_with_user ? ` (Owner: ${list.shared_list_owner})` : "";
+  const numberSharedWith = list.number_shared_with;
+  const sharingText = (numberSharedWith > 0 && list.shared_list_owner == (props.user.first_name + " " + props.user.last_name)) ? ` (Shares: ${numberSharedWith})` : "";
```

Comment:
The second half of this conditional was a smell to me, and it shouldn't be needed.  If the list isn't shared with the user, then the list is owned by the user, therefore you can check the share count and the ownership state to do the same thing without conditional logic.
```suggestion
  const sharingText = (numberSharedWith > 0 && !list.is_shared_with_user) ? ` (Shares: ${numberSharedWith})` : "";
```